streamer/src/nonblocking/quic.rs

```diff
@@ -473,10 +524,48 @@ impl ConnectionTable {
         num_pruned
     }
 
+    fn connection_stake(&self, index: usize, default: u64) -> u64 {
```

Comment:
I think this function should return `None` if the index was invalid, the caller can unwrap to 0